AzureSDK.xcworkspace/xcshareddata/xcschemes/all.xcscheme

```diff
@@ -146,6 +146,20 @@
                ReferencedContainer = "container:sdk/storage/AzureStorageBlob/AzureStorageBlob.xcodeproj">
             </BuildableReference>
          </BuildActionEntry>
+         <BuildActionEntry
```

Comment:
I noticed the Chat target was not added to `all` scheme so it wasn't building. 
I added it here